### PR TITLE
[O2B-1186] QC flag type creation page improvements

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [ ] issue "Status" is set to "In review"
- [ ] PR labels are selected

Notable changes for users:
- NA

Notable changes for developers:
- Fixed QC flag creation component scaling

Changes made to the database:
- NA

### CODE REVIEW

File: lib/public/app.css [hunk 1]
```diff
@@ -81,6 +81,10 @@ html, body {
     width: unset;
 }
 
+.mw-large {
+    max-width: var(--ui-component-large);
+}
+
```
Comment (martinboulais) [inline lib/public/app.css:87]: If you look at line 52, you'll see that:

- fixed width are put before percentage width, I would then put this line at l. 74
- we use `l` not `large`

File: lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ import { LabelPanelHeaderComponent } from '../../../components/common/panel/Labe
 const qcFlagTypeCreationComponent = (qcFlagTypeCreationModel) => {
     const { name, method, color, bad } = qcFlagTypeCreationModel.formData;
 
-    const panel = [
+    return h('.flex-column.gr3.mw-large', [
```
Comment (martinboulais) [inline lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js:29]: Why gr3? You can directly use g3

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/public/app.css (hunk 1 above)
Comment (xsalonx) [reply lib/public/app.css:87]: applied

File: lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js (hunk 2 above)
Comment (xsalonx) [reply lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js]: applied

**Review by martinboulais — APPROVED** [review]

**Pull request merged**